packages/app/backend-http-client/src/client/httpClient.ts

```diff
@@ -93,7 +93,7 @@ export async function sendDelete<
       reset: options.disableKeepAlive ?? false,
       bodyTimeout: Object.hasOwn(options, 'timeout') ? options.timeout : DEFAULT_OPTIONS.timeout,
       headersTimeout: Object.hasOwn(options, 'timeout') ? options.timeout : DEFAULT_OPTIONS.timeout,
-      throwOnError: false,
+      throwOnError: undefined,
```

Comment:
undici no longer supports this as a parameter and expects https://undici.nodejs.org/#/docs/api/Dispatcher?id=responseerror to be used instead.

Since we always consumed this parameter directly instead of relying on undici handling (in order to provide sufficient amount of error details), we just ensure that we no longer pass-through this parameter.